### PR TITLE
Server docs re-structure

### PR DESCRIPTION
**Pull request by gilgongo**
# Changes

(yet another huge) Restructure to place headless Linux as the most common method of running a server. Demote bandwidth and QoS sections to child pages. Tweaks to headings and some minor additional wording. 

## Does this need translation?

- [x] Yes 
- [ ] No

### CODE REVIEW

File: wiki/en/Running-a-Server.md [hunk 1]
```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
+
+The following minimum setup is required to [run a Registered Server](Running-a-Server#server-types):
+
+~~~
+jamulus --nogui --server \
+        --directoryserver genreServer:port \
+        --serverinfo "yourServerName;yourCity;[country ID]"
+~~~
+
+**Note**: Semicolon and newline characters are not allowed in `--serverinfo`
```
Comment (pljones) [inline wiki/en/Running-a-Server.md]: > not allowed in `--serverinfo`

Except the text above that clearly has semicolons.  What it should say is

> not allowed in `yourServerName` and `yourCity` within the `--serverinfo` argument

File: wiki/en/Running-a-Server.md [hunk 2]
```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
+
+Replace `genreServer:port` in the example above with one of the following options:
```
Comment (pljones) [inline wiki/en/Running-a-Server.md]: > Replace `genreServer:port` in the example above with one of the following options:

Should say:
> To register with one of the Directories built into the Jamulus Client, replace `genreServer:port` in the example above with one of the following options:

After the list, it should mention about Custom Directories - they are used in the same way from the command line, providing the Server Address in the same format.

File: wiki/en/Running-a-Server.md [hunk 3]
```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
+
+
+| Genre |   Server address           |
```
Comment (pljones) [inline wiki/en/Running-a-Server.md]: Capital "A" on "address" ... if I get my way...

File: wiki/en/Running-a-Server.md [hunk 4]
```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
+
+#### Running as a Directory
+
+If you wish to run a [Custom Directory](Running-a-Server#3-custom-directory) please see [this guide](Custom-Directories).
+
+### Maintainance
+
+#### Upgrading
+
+To upgrade your Server to a newer version, simply download a new .deb and re-install as step 3.
```
Comment (pljones) [inline wiki/en/Running-a-Server.md]: "step 3" here refers back to the list in the _previous_ heading 3 section - at least link to that heading.

File: wiki/en/Running-a-Server.md [hunk 5]
```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
+ [Service]
+ Type=oneshot
+ ExecStart=/bin/systemctl kill -s SIGUSR1 Jamulus-Server
+~~~
+
+_Note: The Jamulus service name in the `ExecStart` line needs to be the same as the `.service` file name you created when setting systemd to control your Jamulus Server. So in this example it would be `Jamulus-Server.service`_
+
+Run `sudo systemctl daemon-reload` to register them for first use.
+
+Now you can run these with the `service start` command, for example:
```
Comment (pljones) [inline wiki/en/Running-a-Server.md]: Shouldn't be using the `service` command, it's deprecated.  `sudo service (something) start` should be replaced with `sudo systemctl start (something)`.

File: wiki/en/Running-a-Server.md [hunk 6]
```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
+# Servers on the desktop
+
+Jamulus can be configured to run in server mode on the desktop. This gives you a graphical user interface to control most of the settings. 
 
 * **Windows users** - Use the "Jamulus Server" icon in the Windows Start menu.
 
 * **macOS users** - Double-click the "Jamulus Server" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).
 
-* **Linux users** - Open a terminal window (`CTRL+ALT+t` on Ubuntu and related distros) and type `jamulus -s`. Hit return and you should see the Server control window. You can stop the Server by closing the Server window, or by using `CTRL+C` in the terminal. _(To run a headless Server [read this guide](Server-Linux))_
+* **Linux users** - Open a terminal window (`CTRL+ALT+t` on Ubuntu and related distros) and type `jamulus -s`. Hit return and you should see the Server control window. You can stop the Server by closing the Server window, or by using `CTRL+C` in the terminal. _(To run a headless Server [read this guide](#installing-and-running-a-server))_
```
Comment (pljones) [inline wiki/en/Running-a-Server.md]: Why don't we provide a `.desktop` file for this?

Comment (gilgongo) [reply wiki/en/Running-a-Server.md]: Guess we should - so Client and Server `.desktop` icons like on macOS you mean? I suppose the only reason we haven't is that the number of people running desktop servers is probably next to zero.

File: wiki/en/Running-a-Server.md [hunk 7]
```diff
@@ -118,9 +118,9 @@ jamulus --nogui --server \
         --serverinfo "yourServerName;yourCity;[country ID]"
 ~~~
 
-**Note**: Semicolon and newline characters are not allowed in `--serverinfo`
+**Note**: Semicolon and newline characters are not allowed in `yourServerNam`e and `yourCity` within the `--serverinfo` argument
```
Comment (pljones) [inline wiki/en/Running-a-Server.md]: Typ`o

File: wiki/en/Running-a-Server.md (hunk 6 above)
Comment (pljones) [reply wiki/en/Running-a-Server.md]: Yep - desktop linux is probably a pretty small percentage of desktop users (I do wish we had the download stats easily accessible for each of the release files).  But it does then mean we can treat all three platforms more equally in the desktop case, without having to force Linux desktop users onto the command line.

("Tricks and Tips" should have "how to set up additional desktop shortcuts for additional profiles" - i.e. using `--inifile` - covering all platforms, too.)

Comment (jujudusud) [reply wiki/en/Running-a-Server.md]: Why "Ubuntu and related distros" ? Is it possible to write "Debian and related distros" instead ?
@pljones , ArchLinux and LibraZiK3 are usually providing .desktop files with Jamulus.

Comment (pljones) [reply wiki/en/Running-a-Server.md]: @jujudusud We'll be hosting our own repo for the .deb as soon as we can work out how, that's why it's important to make it as complete as possible.  I thought we already provided a _client_ .desktop, anyway.  It's the _server_ one I'm talking about here.

Comment (pljones) [reply wiki/en/Running-a-Server.md]: Agree on the "Debian and ..." point, though.  Get the heritage right.

Comment (ann0see) [reply wiki/en/Running-a-Server.md]: > I do wish we had the download stats easily accessible for each of the release files

Sourceforge + https://somsubhra.github.io/github-release-stats/ should give insight

Comment (gilgongo) [reply wiki/en/Running-a-Server.md]: SF may still be giving reasonably accurate percentages too maybe: https://sourceforge.net/projects/llcon/files/stats/os

**Review by jujudusud — APPROVED** [review]

File: wiki/en/Running-a-Server.md (hunk 6 above)
Comment (jujudusud) [reply wiki/en/Running-a-Server.md]: @pljones, so am I:
Archlinux-desktop

File: wiki/en/Running-a-Server.md [hunk 8]
```diff
@@ -83,48 +83,151 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
+|**Genre Classical/Folk** |`classical.jamulus.io:22524`|
+|**Genre Choral/Barbershop** |`choral.jamulus.io:22724`|
+
+You can also specify a [Custom Directory](#3-custom-directory) in the same way from the command line, providing the Server Address in the same format.
+
+#### Running as a Directory
+
+If you wish to run a [Custom Directory](Running-a-Server#3-custom-directory) please see [this guide](Custom-Directories).
+
+### Maintainance
```
Comment (ignotus666) [inline wiki/en/Running-a-Server.md]: Maint**e**nance

File: wiki/en/Running-a-Server.md [hunk 9]
```diff
@@ -83,48 +83,151 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
+`journalctl -f -u jamulus-headless`
 
-Note also that mean ADSL2 transfer rate is 10 Mbit/s for downstream and 1 Mbit/s for upstream. The actual performance depends on distance to the provider, which may [theoretically range from 24 Mbit/s at 0.3 km to 1.5 Mbit/s at 5.2 km](https://en.wikipedia.org/wiki/Asymmetric_digital_subscriber_line) for download rate.
+#### Recording
 
+When using the recording function with the `-R` command line option, if the Server receives a SIGUSR1 signal during a recording, it will start a new recording in a new Directory. SIGUSR2 will toggle recording enabled on/off.
 
-# Starting a Server
+To send these signals using systemd, create the following two `.service` files in `/etc/systemd/system`, calling them something appropriate (e.g. `newRecording-Jamulus-serv
+er.service`).
```
Comment (ignotus666) [inline wiki/en/Running-a-Server.md]: Space in "server"

**Review by pljones — APPROVED** [review]

**Review by jujudusud — APPROVED** [review]

File: wiki/en/Running-a-Server.md [hunk 10]
```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
+### Installation
 
-## Network bandwidth
+1. Download the [latest headless .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }})
+1. Update apt to make sure you have a current list of standard packages: `sudo apt update`
+1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}`
+1. Enable the headless Server process via systemd: `sudo systemctl enable jamulus-headless`
+1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file in `/lib/systemd/system/jamulus-headless.service` (By default you will be running an Unregistered Server).
+1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
+1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
```
Comment (ann0see) [inline wiki/en/Running-a-Server.md:101]: I still think the installation guide should be moved up a bit.

File: wiki/en/Running-a-Server.md [hunk 11]
```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
-# Bandwidth use
 
-## Audio bandwidth
+# Installing and running a Server 
 
-The audio settings have an impact on the required network bandwidth. The table below summarises network requirements with respect to the configuration of:
-* Channels : stereo/mono
-* Quality : high/medium/low
-* Audio buffer duration : 2.67 ms, 5.33 ms, 10.67 ms, 21.33 ms
+Most people run Jamulus as a "pure" Server on **hardware without audio** (e.g. on a 3rd party/cloud host) running Linux. The following guide assumes you are familiar with the command line and Debian/Ubuntu or similar distribution using systemd. To run a server on Windows or on the desktop with a graphical user interface, [see this section](#servers-on-the-desktop).
```
Comment (ann0see) [inline wiki/en/Running-a-Server.md:89]: To run a server on Windows/macOS on the desktop with a graphical user interface

File: wiki/en/Running-a-Server.md [hunk 12]
```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
+1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
+1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
 
-There is one upstream (musician sending to the Server) and one downstream (server sending back the mix to the musician)
-<figure>
-	<img src="{% include img/en-screenshots/bandwidth-diagram.inc %}" loading="lazy" alt="A diagram of Jamulus network bandwidths from different audio qualities ranging from low to high">
-<figcaption>Calculate bandwidth use </figcaption>
-</figure>
+You may also be interested in downloading [this set of useful tools](https://github.com/jamulussoftware/jamulus/tree/master/tools) from the Jamulus repository (clone the Git repo and also call `git submodule update --init`).
+
```
Comment (ann0see) [inline wiki/en/Running-a-Server.md:104]: I'd remove this.

File: wiki/en/Running-a-Server.md [hunk 13]
```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
...
+# Servers on the desktop
+
+Jamulus can be configured to run in Server mode on the desktop. This gives you a graphical user interface to control most of the settings. 
 
 * **Windows users** - Use the "Jamulus Server" icon in the Windows Start menu.
 
 * **macOS users** - Double-click the "Jamulus Server" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).
 
-* **Linux users** - Open a terminal window (`CTRL+ALT+t` on Ubuntu and related distros) and type `jamulus -s`. Hit return and you should see the Server control window. You can stop the Server by closing the Server window, or by using `CTRL+C` in the terminal. _(To run a headless Server [read this guide](Server-Linux))_
+* **Linux users** - Open a terminal window (`CTRL+ALT+t` on Debian and related distros) and type `jamulus -s`. Hit return and you should see the Server control window. You can stop the Server by closing the Server window, or by using `CTRL+C` in the terminal. _(To run a headless Server [read this guide](#installing-and-running-a-server))_
```
Comment (ann0see) [inline wiki/en/Running-a-Server.md:229]: ... or run the newly created desktop shortcut. Basically it's similar to Windows.

File: wiki/en/Running-a-Server.md (hunk 11 above)
Comment (pljones) [reply wiki/en/Running-a-Server.md:89]: > To run a server on Windows/macOS on the desktop with a graphical user interface

Are you intending to _exclude_ Linux desktop?

Comment (ann0see) [reply wiki/en/Running-a-Server.md:89]: No. Not really.

File: wiki/en/Running-a-Server.md (hunk 10 above)
Comment (gilgongo) [reply wiki/en/Running-a-Server.md:101]: Hard to know where we'd put it...

One thing we **could** do is have it on a separate page, and add that to the main nav in the same way that we do for the client (so "Server Administration" as a child link of "Server setup" perhaps?). The GUI would need to be mentioned there as well though. That might make it look as if you need to run your own server to use Jamulus, but hey maybe that ship's sailed.

File: wiki/en/Running-a-Server.md (hunk 11 above)
Comment (gilgongo) [reply wiki/en/Running-a-Server.md:89]: This is why I purposefully _didn't_ mention the platform :-)

**Pull request merged**